### DISCUSSION TITLE
Imposter - spigotmc

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#370] [Announcements]
Hello everyone,

It has come to my attention that, someone is publishing Eden to [spigotmc](https://www.spigotmc.org/resources/eden.110289/). This is illegal according to our license and please do not download the plugin from there as the plugin might contain virus or such.

The only platform for official download is in github. I didn't post any jar plugins in other site. And I only posted a thread about this plugin in builtbybit.

**Comment by aappaapp** [comment]
Trying to report the plugin

**Reply by GoodestEnglish** [reply]
appreciate it :)

**Reply by OpticFusion1** [reply]
"This is illegal according to our license" 
Actually, given that you're using GPLv3 that's perfectly okay.
https://choosealicense.com/licenses/gpl-3.0/

Edit: The only thing that _would_ get them in trouble on spigot is not having enough significant changes between this version and the one on spigot.
